<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuoneistoDialog</class>
 <widget class="QDialog" name="HuoneistoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Huoneiston tiedot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../pic/pic.qrc">
    <normaloff>:/pic/talo.png</normaloff>:/pic/talo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Huoneistotunnus</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tunnusEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Asiakas</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="AsiakasToimittajaValinta" name="asiakas" native="true">
       <property name="MuokkaaUusi" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="laskuTab">
      <attribute name="icon">
       <iconset resource="../../pic/pic.qrc">
        <normaloff>:/pic/lasku.png</normaloff>:/pic/lasku.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Kuukausilaskutus</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="tuoteSuodatus">
             <property name="placeholderText">
              <string>Suodata tuotteita</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="tuoteView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableView" name="laskuView"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="poistaNappi">
               <property name="text">
                <string>Poista rivi</string>
               </property>
               <property name="icon">
                <iconset resource="../../pic/pic.qrc">
                 <normaloff>:/pic/poistarivi.png</normaloff>:/pic/poistarivi.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="muistiinpanotTab">
      <attribute name="icon">
       <iconset resource="../../pic/pic.qrc">
        <normaloff>:/pic/kommentti-harmaa.png</normaloff>:/pic/kommentti-harmaa.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Muistiinpanot</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="muistiinpanoEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AsiakasToimittajaValinta</class>
   <extends>QWidget</extends>
   <header location="global">rekisteri/asiakastoimittajavalinta.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HuoneistoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HuoneistoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
